packages/tests/src/helpers/assertions.ts

```diff
@@ -0,0 +1,22 @@
+import ScreenshotService from './../services/ScreenshotService'
+
+// TODO - .env
+const SPECTRE_UI_URL='https://spectre.service.bcdev'
+
+// TODO - Add as browser command
+export async function assertElementLooks(browser:WebdriverIO.BrowserObject, selector:string, name:string): Promise<void> {
```

Comment:
Based on your type here, are you supposed to be returning something from this function?

Reply:
✅ Removed the return type because it was a void promise based on advice in this PR